script/scriptMaster.sh

```
@@ -490,7 +490,28 @@ scriptMaster() {
+        "1) All networks" \
+        "2) One specific network (selection in next screen)"
+    )
+    echo "[info] selected option: $SELECTION_NETWORK"
+
+    if [[ "$SELECTION_NETWORK" == "1)"* ]]; then
+      # call parse target state function for all networks
+      parseTargetStateGoogleSpreadsheet "$ENVIRONMENT"
+    else
+      checkNetworksJsonFilePath || checkFailure $? "retrieve NETWORKS_JSON_FILE_PATH"
+      # get user-selected network from list
+      local NETWORK=$(jq -r 'keys[]' "$NETWORKS_JSON_FILE_PATH" | gum filter --placeholder "Network")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)